docker-compose.yml

```
@@ -62,7 +62,7 @@ services:
         condition: service_healthy
 
   kafka:
-    image: bitnami/kafka:latest
+    image: ubuntu/kafka:latest
```

Wrong. Removing this change. 

```suggestion
    image: bitnami/kafka:latest
```